development/includes/sm-share-sidebar.pug

```diff
@@ -0,0 +1,21 @@
+script(src="/resources/js/sm-share-sidebar.js")
+div.sm-share-sidebar
+  a.share-slider-toggle Share me!
+    i(class="fa fa-heart-o")
+  ul.sm-share-buttons
+    //- <!-- Facebook -->
+    a.sm-share-selector(id="facebook-share-button" href="https://www.facebook.com/sharer/sharer.php?u=$PAGE_URL" target="_blank")
```

Comment:
You have this reversed :P 

It should be: 
```
  li
    a.sm-share-selector(id="facebook-share-button" href="https://www.facebook.com/sharer/sharer.php?u=$PAGE_URL" target="_blank")
      i(class=" fa fa-facebook-official fa-3x facebook-box")
```
Inside of a `<ul>` the next element has to be a list item `<li>`